README.md

```diff
@@ -340,6 +340,40 @@ The aforementioned guide works well for UI components. However, components like
    )
 ```
 
+### Managing SVG Icons
+Icons are handy and help build beautiful web pages, but sometimes, including them can decrease performance. There isn't a silver bullet solution that allows them to be added while not decreasing the performance. So, this project suggests using SVGs from a single SVG file, located in `/static/icons/icons.svg`, and loading them with the `IconSVG` component.
+
+So, follow these steps to add or use a new SVG icon:
+
+> :warning: This is a suggestion, evaluate if this fits your scenario.
+
+#### Adding an SVG icon
+1. In the SVG file, change the `svg` tag to `symbol`.
+2. Add an `id` to the symbol. Remember to use an unique `id` and do not replicate it.
```

Comment:
Add a code snippet illustrating an example of how the `id` looks in the SVG file. Something similar to the example in *step 2* of the next section, *Using an SVG icon*.

Reply:
Ok, good catch.